packages/next/build/webpack/plugins/font-stylesheet-gathering-plugin.ts

```diff
@@ -4,7 +4,6 @@ import {
   isWebpack5,
   sources,
 } from 'next/dist/compiled/webpack/webpack'
-import { namedTypes } from 'ast-types'
```

Comment:
I think you can leave this in as a devDependency only and use

```
import type { namedTypes } from 'ast-types'
```

Reply:
Unfortunately not because of the way `.d.ts` is generated for some files which then crashes TS checking if the dep is not there even when not used.